common/djangoapps/third_party_auth/config/waffle.py

```diff
@@ -18,8 +17,21 @@
 # .. toggle_target_removal_date: 2021-04-31
 # .. toggle_warnings: None.
 # .. toggle_tickets: ENT-4034
-ENABLE_MULTIPLE_SSO_ACCOUNTS_ASSOCIATION_TO_SAML_USER = LegacyWaffleSwitch(
-    _WAFFLE_SWITCH_NAMESPACE,
-    'enable_multiple_sso_accounts_association_to_saml_user',
-    __name__
+ENABLE_MULTIPLE_SSO_ACCOUNTS_ASSOCIATION_TO_SAML_USER = WaffleSwitch(
+    f'{WAFFLE_NAMESPACE}.enable_multiple_sso_accounts_association_to_saml_user',
+    module_name=__name__
```

Comment:
@nizarmah Thank you for taking care of this. I am creating another PR for this waffle switch change. Also one NIT here, we are declaring the variable in the string but not passing value to it. I think it should be
`'{WAFFLE_NAMESPACE}.enable_multiple_sso_accounts_association_to_saml_user'.format(WAFFLE_NAMESPACE=WAFFLE_NAMESPACE)`

Reply:
@zamanafzal the f-strings in python is really helpful in that regard. We don't need to add the `.format` part 😃